core/templates/dev/head/domain/exploration/ExplorationObjectFactory.js

```diff
@@ -23,6 +23,8 @@ oppia.factory('ExplorationObjectFactory', [
   function(
       INTERACTION_SPECS, INTERACTION_DISPLAY_MODE_INLINE, StateObjectFactory,
       UrlInterpolationService) {
+    var currentExploration = null;
```

Comment:
This isn't good practice. Object factories should not be storing state. If you need to store state, you should have a centralized state service (see the simple editor or collection editor for examples).